test/login.js

```diff
@@ -0,0 +1,62 @@
+// Mock a successful login and populate basic profile data for user
+
+// Force fake Login
+
+TEST.$rootScope.userExists = true;
+
+// Logged in user data
+
+// TEST.$rootScope._user = {
```

Comment:
That's a partially working mock for login, but I guess I could trash it since it's not being used.